paddle/framework/operator.h

```diff
@@ -319,5 +319,14 @@ class OperatorWithKernel : public OperatorBase {
   virtual void InferShape(const InferShapeContext& ctx) const = 0;
 };
 
+namespace op_helpers {
+
+// Generate the gradient variable's name of a forward varialbe
+inline std::string GenGradName(const std::string& var) {
```

Comment:
@Superjom @Canpio 

Rename `GRAD_VAR_SUFFIX` --> `GenGradName()`.
```cpp
string GenGradName(string name) {
  return name + "@GRAD";
}
```

and suffix could be `GenGradName("");`